lib/livebook_web/components/core_components.ex

```diff
@@ -739,4 +740,94 @@ defmodule LivebookWeb.CoreComponents do
   def hook_prop(value) do
     Jason.encode!(value)
   end
+
+  @doc ~S"""
+  Renders a table with generic styling.
+
+  ## Examples
+
+      <.table id="users" rows={@users}>
+        <:col :let={user} label="id"><%= user.id %></:col>
+        <:col :let={user} label="username"><%= user.username %></:col>
+      </.table>
+  """
+  attr :id, :string, required: true
+  attr :rows, :list, required: true
+  attr :row_id, :any, default: nil, doc: "the function for generating the row id"
+  attr :row_click, :any, default: nil, doc: "the function for handling phx-click on each row"
+
+  attr :row_item, :any,
+    default: &Function.identity/1,
+    doc: "the function for mapping each row before calling the :col and :action slots"
+
+  attr :show_action_label, :boolean, default: false
```

Comment:
Remove this attr for now. You can consider that it is alwaus true (and always show the label).